IntegrationTests/UI.IntegrationTests/GitUICommands/RunCommandTests.cs

```diff
@@ -361,6 +366,31 @@ public void RunCommandBasedOnArgument_unsupported(string command)
                 });
         }
 
+        [Test]
+        [Description("Makes sure getting a git config value does not raise an exception "
+            + "and even if config value is not set it will just return a blank value")]
```

Comment:
```suggestion
        [Description("Makes sure getting a git config value does not raise an exception and even if config value is not set it will just return a blank value")]
```